install/ossm-installing-openshift-service-mesh.adoc

```diff
@@ -16,6 +16,9 @@ include::modules/ossm-creating-istio-project-using-console.adoc[leveloffset=+2]
 include::modules/ossm-creating-istio-resource-using-console.adoc[leveloffset=+2]
 include::modules/ossm-creating-istio-cni-project-using-console.adoc[leveloffset=+2]
 include::modules/ossm-creating-istiocni-resource-using-console.adoc[leveloffset=+2]
+include::modules/ossm-scoping-service-mesh-with-discoveryselectors.adoc[leveloffset=+1]
+include::modules/ossm-about-discoveryselectors.adoc[leveloffset=+2]
+include::modules/ossm-using-discoveryselectors-scope-service-mesh.adoc[leveloffset=+2]
 include::modules/ossm-about-bookinfo-application.adoc[leveloffset=+1]
 include::modules/ossm-deploying-bookinfo-application.adoc[leveloffset=+2]
```

Comment:
Example lead-in resource
```suggestion
include::modules/ossm-using-discoveryselectors-scope-service-mesh.adoc[leveloffset=+2]
[role="_additional-resources"]
.Additional resources
* xref:../../../installing/installing_vsphere/installation-config-parameters-vsphere.adoc#installation-config-parameters-vsphere[Installation configuration parameters]


include::modules/ossm-about-bookinfo-application.adoc[leveloffset=+1]
include::modules/ossm-deploying-bookinfo-application.adoc[leveloffset=+2]
```

Reply:
Wow @eromanova97! Thank you so much for helping me out with this.